projects/rails-backend/db/migrate/20210804100926_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :line_id
```

Comment:
line_idを持たないユーザーを保存することはないので、NOT NULL制約をつけておうた方がよさそうですね :eyes:

```suggestion
      t.string :line_id, null: false
```

Reply:
line_id はユーザー固有のユニークな値になるはずなので、unique 制約を張っておきたいです
https://qiita.com/NedzumiNeko/items/bf8ad819fdbf087da947